ansible_ai_connect/main/tests/test_views.py

```diff
@@ -339,10 +341,21 @@ def test_chatbot_view_with_rh_user(self):
         self.assertContains(r, TestChatbotView.CHATBOT_PAGE_TITLE)
         self.assertContains(r, self.rh_user.username)
         self.assertContains(r, '<div id="debug" hidden>false</div>')
+        self.assertContains(r, '<div id="stream" hidden>false</div>')
 
     @override_settings(CHATBOT_DEBUG_UI=True)
     def test_chatbot_view_with_debug_ui(self):
         self.client.force_login(user=self.rh_user)
         r = self.client.get(reverse("chatbot"), {"debug": "true"})
         self.assertEqual(r.status_code, HTTPStatus.OK)
         self.assertContains(r, '<div id="debug" hidden>true</div>')
+
+    def test_chatbot_view_with_streaming_enabled(self):
+        llm: ModelPipelineChatBot = apps.get_app_config("ai").get_model_pipeline(
+            ModelPipelineChatBot
+        )
+        llm.config.stream = True
+        self.client.force_login(user=self.rh_user)
+        r = self.client.get(reverse("chatbot"), {"stream": "true"})
```

Comment:
Oh, BTW this may need changing.

See @ldjebran PR about versioning tests.

Reply:
https://github.com/ansible/ansible-ai-connect-service/pull/1512

Reply:
The chatbot is not part of the api versioning as it sits in the root path e.g. /chatbot/

Reply:
Ah, OK.. I was getting my `/chat` confused with my `/chatbot` :smile: 

`/chat` is included in both the `v0` and `v1` URLs.
- https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/ai/api/versions/v0/ai/urls.py#L27
- https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/ai/api/versions/v1/ai/urls.py#L26

`/chatbot` is not:
- https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/main/urls.py#L82